templates/CRM/Admin/Form/ScheduleReminders.tpl

```diff
@@ -280,6 +276,15 @@
           showSaveUpdateChkBox('SMS');
         }
       }
+	  
+	  //CRM-14097 - When un-checking the "send email" box, the email fields do not get hidden
+	  cj(function ( $ ) {
+		$(document).ready(function(){
```

Comment:
These two wrappers are not necessary - this code is already wrapped by `CRM.$(function($) {` on line 178.